metrics/collectors/images/images_collector.py

```diff
@@ -92,6 +94,11 @@ def collect(self):
                 # the path ends with the image filename
                 image_name = path_table[-1]
 
+                image_type = None
+                if "daily-preinstalled" in path:
+                    image_type = "preinstalled"
+                elif "daily-live" in path:
+                    image_type = "live"
```

Comment:
Checking for these strings anywhere in path seems fragile to me. Could we use a more specific check e.g. something like path[99]=='daily-preinstalled'?